src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -0,0 +1,74 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ert._c_wrappers.config import ConfigParser
+
+
+class DeprecationMigrationSuggester:
+    def __init__(self, parser: "ConfigParser"):
+        self._parser = parser
+        self._add_deprecated_keywords_to_parser()
+
+    REPLACE_WITH_GEN_KW = ["RELPERM", "MULTZ", "EQUIL", "GEN_PARAM"]
+
+    def _add_deprecated_keywords_to_parser(self):
+        for kw in self.REPLACE_WITH_GEN_KW:
+            self._parser.add(kw)
+        self._parser.add("UMASK")
+        self._parser.add("HAVANA_FAULT")
+        self._parser.add("MULTFLT")
+        self._parser.add("REFCASE_LIST")
+        self._parser.add("RFTPATH")
+        self._parser.add("END_DATE")
+
+    def suggest_migrations(self, filename: str):
+        suggestions = []
+
+        content = self._parser.parse(filename)
+        for kw in self.REPLACE_WITH_GEN_KW:
+            if content.hasKey(kw):
+                suggestions.append(
+                    "The keyword {kw} was deprecated in 2009 in favor of using"
+                    " GEN_KW and FORWARD_MODEL."
+                )
+        if content.hasKey("UMASK"):
+            suggestions.append(
+                f"The UMASK keyword no longer has any effect "
```

Comment:
github actions: f-string is missing placeholders

Reply:
Fixed